jobs/suite/default.env

```diff
@@ -1,2 +1 @@
-GINKGO_TEST_ARGS=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]
 GINKGO_PARALLEL=y
```

Comment:
the file only contains `GINKGO_PARALLEL=y` now.. maybe have a `--parallel=true` flag instead? thoughts?